### PR TITLE
RFC 2873 (asm!): Allow multiple template string arguments

### PR DESCRIPTION
**Pull request by joshtriplett**
Interpret them as newline-separated.

### CODE REVIEW

File: text/0000-inline-asm.md [hunk 1]
```diff
@@ -372,17 +380,19 @@ reg_operand := dir_spec "(" reg_spec ")" operand_expr
...
-asm := "asm!(" format_string *("," [ident "="] operand) ["," options] [","] ")"
+asm := "asm!(" format_string *("," format_string) *("," [ident "="] operand) ["," options] [","] ")"
 ```
 
 The macro will initially be supported only on ARM, AArch64, x86, x86-64 and RISC-V targets. Support for more targets may be added in the future. The compiler will emit an error if `asm!` is used on an unsupported target.
 
 [format-syntax]: https://doc.rust-lang.org/std/fmt/#syntax
 
-## Template string
+## Template strings
```
Comment (Amanieu) [inline text/0000-inline-asm.md]: The terminology is wrong here: there is a single template string, which can be composed from multiple string literals.

Comment (joshtriplett) [reply text/0000-inline-asm.md]: Fixed.

**Comment by Amanieu** [comment]
Can you update this with the latest changes in the other PR?

**Comment by joshtriplett** [comment]
@Amanieu Done.

**Pull request merged**